source/Extension.svelte

```diff
@@ -49,8 +49,8 @@
 	<button type="button" class="ext-name" on:click={toggleExtension} on:contextmenu>
 		<img alt="" src={getIcon(icons, 16)}>{name}
 	</button>
-		{#if  showExtras || (optionsUrl && enabled)}
-			<a href='chrome://extensions/?options={id}' title={getI18N('gotoOpt')} on:click={openInTab}>
+		{#if optionsUrl && enabled}
```

Comment:
I think this is the right logic
```suggestion
		{#if optionsUrl && (showExtras || enabled)}
```

Reply:
- the current logic in the online version for the options icon is not correct, extensions without options page should not show options icon.
- disabled extensions, even with options page, should not show the icon either, because click on it would open an error page. 
- with my change, other icons would still be shown when `showExtras`

Reply:
I understand the issue, but how can they be shown if there’s no `showExtras` in this condition?

Reply:
You’re right, it’s all good